### PR TITLE
Hermes integration

### PR DESCRIPTION
**Pull request by ar4s**
Related to https://github.com/mkurek/pyhermes/pull/1

### CODE REVIEW

File: requirements/test.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 -e .
 -r prod_ldap.txt
+-r hermes.txt
```
Comment (mkurek) [inline requirements/test.txt]: why not in base.txt?

File: src/ralph/settings/base.py [hunk 2]
```diff
@@ -297,3 +297,11 @@ def os_env_true(var, default=''):
     INSTALLED_APPS += (
         'ralph.dns',
     )
+
+
+ENABLE_HERMES_INTEGRATION = os.environ.get('ENABLE_HERMES_INTEGRATION', False)
```
Comment (mkurek) [inline src/ralph/settings/base.py]: please add here `HERMES = {'ENABLED': False}` too - without this pyhermes will try to publish events

Comment (ar4s) [inline src/ralph/settings/base.py]: done

Comment (mkurek) [inline src/ralph/settings/base.py]: use `os_env_true` here

**Comment by mkurek** [comment]
:ok:

File: Makefile [hunk 3]
```diff
@@ -22,7 +22,7 @@ install: install-js
 	pip3 install -r requirements/prod.txt
 
 install-test:
-	pip3 install -r requirements/test.txt
+	pip3 install --no-cache -r requirements/test.txt
```
Comment (ghost) [inline Makefile]: why this? :)
and
did you mean
`--no-cache-dir`?

File: src/ralph/urls/base.py [hunk 4]
```diff
@@ -28,6 +28,7 @@
     url(r'^', include(admin.urls)),
     url(r'^api/', include(api_urls)),
     url(r'^api-token-auth/', views.obtain_auth_token),
+    url(r'^hermes/', include('pyhermes.apps.django.urls')),
```
Comment (ghost) [inline src/ralph/urls/base.py]: shouldnt this be switchable?

Comment (szok) [inline src/ralph/urls/base.py]: 👍

**Comment by ghost** [comment]
:ok:

**Pull request merged**